src/components/MainTabs/MainTabs.test.tsx

```diff
@@ -22,6 +22,11 @@ test("renders without crashing", () => {
   expect(container).toBeInTheDocument();
 });
 
+test("renders correctly", () => {
```

Comment:
you could explicitly  test for what you expect on the navbar. Since those are specifications